### DISCUSSION TITLE
Light browser lib to decode server response ?

### DISCUSSION THREAD

**Discussion by Mupli** [e3dio/packBytes#13] [Ideas]
It would be nice to return this data for browser. With 50x smaller size it could be super fast for example socket and chat responses.  or even encoding and ajax decoding html on browser side.

**Comment by e3dio** [comment]
This library runs in both Node.js and Web Browser, I use it to send websocket messages between server and browser. I need to update the docs to better explain how to do that, docs are old

**Reply by Mupli** [reply]
yes please.  I expected to have something like browser install info .. 

```html
<script src="...packbytes.js"></script> 
```
and use  <script >**fetch** example and **websocket** </script>
